app/manifest/v2/firefox.json

```diff
@@ -4,5 +4,20 @@
       "id": "webextension@metamask.io",
       "strict_min_version": "78.0"
     }
-  }
+  },
+  "permissions": [
+    "storage",
+    "unlimitedStorage",
+    "clipboardWrite",
+    "http://localhost:8545/",
+    "https://*.infura.io/",
+    "https://*.codefi.network/",
+    "https://chainid.network/chains.json",
+    "https://lattice.gridplus.io/*",
+    "activeTab",
+    "tabs",
```

Comment:
Note: Only firefox would be needing this, and can be removed after updating to >85 version.
For chrome, we won't include in permissions, can see the issue https://bugs.chromium.org/p/chromium/issues/detail?id=661311#c1